style.css

```diff
@@ -470,3 +482,380 @@ h2 {
+    overflow-x: hidden;
+  }
+
+  .links {
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    gap: 24px;
+  }
+
+  .dropdown {
+    display: flex;
+    flex-direction: row;
+    margin-right: 156px;
+    padding-bottom: 30px;
+  }
+
+  a {
+    text-decoration: none;
+    color: #fff;
+    width: 63px;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .mail {
+    display: block;
+    width: 24px;
+    height: 24px;
+    background-image: url(./Images/Icon\ -\ Mail.png);
+    margin-left: 20px;
+    margin-top: 20px;
+    gap: 30px;
+  }
+
+  header {
+    width: 100%;
+    height: 8px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .headline {
+    width: 100%;
+    height: 550px;
+    background-image: url(/Images/Illustration-header@2x.png);
+    background-size: 500px 768px;
+    background-position: 100% 30%;
+  }
+
+  .brands {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 20px;
+    height: 98.96px;
+    margin-top: 150px;
+  }
+
+  .icon {
+    display: block;
+    padding: 10px;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 192px 0;
+    position: absolute;
+    width: 566px;
+    height: 500px;
+    left: 143px;
+    top: 72px;
+  }
+
+  .headi {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-left: 235px;
+    width: 566px;
+  }
+
+  .title {
+    display: block;
+    padding: 0;
+    margin-left: 220px;
+    width: 556px;
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: auto;
+  }
+
+  p {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 0;
+  }
+
+  .About-myself {
+    width: 100%;
+    height: 600px;
+  }
+
+  .Recent-work {
+    height: 52px;
+    width: 100%;
+    margin-top: 0;
+  }
+
+  .middle {
+    display: flex;
+    flex-direction: row;
+    margin-left: 150px;
+    gap: 24px;
+  }
+
+  .about-img {
+    background-image: url(./Images/Img-Placeholder4.png);
+    height: 448px;
+    width: 684px;
+  }
+
+  .rightblock {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 14px;
+    height: 384px;
+    margin-top: 50px;
+  }
+
+  .Multi-Post-ul {
+    height: 40px;
+    width: 297px;
+  }
+
+  .work-container {
+    height: 448px;
+    width: 100%;
+    border-radius: 0;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    gap: 20px;
+    justify-items: center;
+    align-items: center;
+  }
```

Comment:
- [ ] Nice one with the work section. 👏 . For final touches, please adjust the following:
-  The buttons should only appear when there's a hover, right now, yours is visible. kindly fix this 👍 
- Secondly, on hover, the cards should change to a white background. I've attached the figma variant for your review.
- [ ] You've really put in the work, but these changes are needed to make the work look professional, and make you a better developer.

## Figma:
![image](https://user-images.githubusercontent.com/50990925/222881145-8cdd3201-bd21-4008-ad2e-788393bb4650.png)

## Your design:
![image](https://user-images.githubusercontent.com/50990925/222881150-c0c5c620-ee20-4ab4-b023-2eca875c0913.png)

Reply:
Hi @Shedrack-Sunday  any idea of how i can implement this?